README.md

```diff
@@ -39,7 +46,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Github page for this project
 
-- https://fernando-silvabr66.github.io/MyPortfolio-W3/  <!--- Review this line whrn it´s the apropriate moment -->
+- https://fernando-silvabr66.github.io/webdevweek/
 
 
```

Comment:
- [ ] OPTIONAL: Please in the Acknowledgements" section give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Done. I had really forgotten to mention that. Thanks.